ui/components/requirements/requirement.jsx

```diff
@@ -15,13 +16,37 @@ export default function Requirement({ requirement }) {
       <div className="req-text col col-10">
         { reqTexts }
         <div className="clearfix mt3">
-          <div className="applies-to mr2">
+          <div className="applies-to mr2 metadata">
             Applies to: [not implemented]
           </div>
-          <div className="sunset-date">
-            Sunset date by { requirement.policy.sunset || 'none' }
+          { requirement.issuing_body ? (
+            <div className="issuing-body metadata">
+              Issuing body: { requirement.issuing_body }
+            </div>
+          ) : (
+            ''
+          ) }
```

Comment:
That makes sense, but I'm unclear on the right way to do this—it seems slightly odd for a const to have a full element in it. Is there an equivalent here to the Python approach of something like this?

```
element = """
<div className="{0} metadata">
    {1}: {2}
</div>
"""

element.format("issuance", "Policy issuance", requirement.policy.omb_policy_id)
```